src/api/operator/operatorApi.ts

```diff
@@ -43,6 +33,22 @@ function _getApiBaseUrl(networkId: Network): string {
   }
 }
 
+/**
+ * Given a list of (key,value), return a URL query string prepended by `?`
+ * Filters out values that are undefined. Empty string is accepted
+ *
+ * @param params List of key value pairs
+ */
+function buildQueryString(params: [key: string, value?: string][]): string {
```

Comment:
is this sth the `URL` or `URLSearchParams` apis can do? if not then ignore me

Reply:
Not sure, but yeah, good point. I thought I saw this somewhere before...
Here I'm building the URL query string (`?param=value&param=value`), and for sure there are things for building that already.
Will look it up